config/jobs/kubernetes-sigs/node-feature-discovery/node-feature-discovery-presubmits.yaml

```diff
@@ -43,3 +43,22 @@ presubmits:
         - runner.sh
         args:
         - scripts/test-infra/build-image.sh
+  - name: pull-node-feature-discovery-build-gh-pages
+    always_run: true
+    decorate: true
+    annotations:
+      testgrid-dashboards: sig-node-node-feature-discovery
+      testgrid-tab-name: build-gh-pages
+      description: "Test build of gh-pages documentation"
```

Comment:
Why does this require kind and/or a privileged security context?

Reply:
Thanks for the review @fejta !

Kind is certainly not needed and was a copy-paste mistake. This is now fixed in the new version of the PR. Also dropped kind from the build-image job. Docs are built in docker, thus (AFAIU) dind and privileged context is needed. Please correct me if I'm wrong or if some other image than kubekins would be more appropriate.